lib/Sema/TypeCheckDeclOverride.cpp

```diff
@@ -970,6 +970,17 @@ static GenericSignature *getOverrideGenericSignature(ValueDecl *base,
   }
   unsigned derivedDepth = 0;
 
+  auto key = swift::ASTContext::OverrideSignatureKey(
```

Comment:
Please move getOverrideGenericSignature() to a method on ASTContext. See for example ASTContext::getExistentialSignature()